src/entt/meta/meta.hpp

```diff
@@ -1409,6 +1525,23 @@ class meta_type {
 }
 
 
+template<typename... Args>
+meta_any meta_any::invoke(const id_type id, Args &&... args) const {
```

Comment:
Out of curiosity, why an out-of-line definition?

Reply:
`meta_type` is defined after `meta_any` so I can't reference it in inline definition.